content/en/docs/components/katib/katib-config.md

```diff
@@ -0,0 +1,211 @@
++++
+title = "Katib Configuration Overview"
+description = "How to make changes in Katib configuration"
+weight = 40
+                    
++++
+
+This page describes information about
+[Katib config](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/katib-controller/katib-config.yaml).
+
+Katib config is the Kubernetes
+[Config Map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) that contains information about:
+
+1. Current [metrics collectors](/docs/components/katib/experiment/#metrics-collector) (`key = metrics-collector-sidecar`)
+1. Current [algorithms](/docs/components/katib/experiment/#search-algorithms-in-detail) (suggestions) (`key = suggestion`).
+
+Katib Config Map must be deployed in
+[`KATIB_CORE_NAMESPACE`](/docs/components/katib/env-variables/#katib-controller)
+namespace with `katib-config` name. Katib controller parses Katib config when you submit Experiment.
+
+You can edit this Config Map even after deploying Katib.
+
+If you deploy Katib in Kubeflow namespace, to edit Katib config run this:
+
+`kubectl edit configMap katib-config -n kubeflow`
+
+## Metrics Collector Sidecar settings
+
+These settings are related to Katib metrics collectors, where:
+
+- key = `metrics-collector-sidecar`
+- value = corresponding JSON settings for each metrics collector kind.
+
+Example for the `File` metrics collector with all settings:
+
+```json
+metrics-collector-sidecar: |-
+{
+  "File": {
+    "image": "docker.io/kubeflowkatib/file-metrics-collector",
+    "imagePullPolicy": "Always",
+    "resources": {
+      "requests": {
+        "memory": "200Mi",
+        "cpu": "250m",
+        "ephemeral-storage": "200Mi"
+      },
+      "limits": {
+        "memory": "1Gi",
+        "cpu": "500m",
+        "ephemeral-storage": "2Gi"
+      }
+    }
+  },
+  ...
+}
+```
+
+All of these settings except **`image`** can be omitted. If you don't specify any other settings, default value is set.
+
+1. `image` - Docker image for the `File` metrics collector's container.
```

Comment:
```suggestion
1. `image` - a Docker image for the `File` metrics collector's container.
```

Reply:
Agree, changed.
@8bitmp3 I made all list items with not capital letter to be consistent (e.g. https://github.com/kubeflow/website/blame/62aba5ce7a065eb2b60d6b7884f4ab075be33fcc/content/en/docs/components/katib/katib-config.md#L66)
WDYT ?